### PR TITLE
Adding skore

### PR DESCRIPTION
**Pull request by glemaitre**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tuscland** [comment]
Please add retromechs to the list of maintainers.

**Comment by augustebaum** [comment]
Opting in

File: recipes/skore/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  license_family: MIT
+  license_file: LICENSE
+  doc_url: https://probabl-ai.github.io/skore/
+  dev_url: https://github.com/probabl-ai/skore
+
+extra:
+  recipe-maintainers:
+    - augustebaum
+    - glemaitre
+    - MarieS-WiMLDS
```
Comment (glemaitre) [inline recipes/skore/meta.yaml:57]:
```suggestion
    - MarieS-WiMLDS
    - retromechs
```

**Comment by thomass-dev** [comment]
Opt in.

**Comment by tuscland** [comment]
I opt in

**Comment by sylvaincom** [comment]
I opt in

**Comment by MarieS-WiMLDS** [comment]
opting in too!

File: recipes/skore/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    - retromechs
+    - rouk1
+    - sylvaincom
+    - thomass-dev
+    - tuscland
```
Comment (glemaitre) [inline recipes/skore/meta.yaml:62]:
```suggestion
    - augustebaum
    - glemaitre
    - MarieS-WiMLDS
    - sylvaincom
    - thomass-dev
    - tuscland
```

File: recipes/skore/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+{% set name = "skore" %}
+{% set version = "0.3.0" %}
```
Comment (glemaitre) [inline recipes/skore/meta.yaml:2]:
```suggestion
{% set version = "0.4.1" %}
```

File: recipes/skore/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+{% set sha256 = "841f152c34d5bdd59cfa94cde63de4add35fa7effaa470b80939a9848ed16c37" %}
```
Comment (glemaitre) [inline recipes/skore/meta.yaml:3]:
```suggestion
{% set sha256 = "a24410ffb26b461c9b48d40c703242e4015b49ae01e73a91163d0df55d9100c1" %}
```

File: recipes/skore/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - diskcache
+    - fastapi
+    - plotly >=5,<6
+    - rich
+    - skops
+    - uvicorn
```
Comment (glemaitre) [inline recipes/skore/meta.yaml:31]:
```suggestion
    - uvicorn
    # remove in 0.5.0
    - pandas
```

File: recipes/skore/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+    # remove in 0.5.0
+    - pandas
```
Comment (glemaitre) [inline recipes/skore/meta.yaml:33]:
```suggestion
    - pandas
    - pyarrow
```

File: recipes/skore/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (glemaitre) [inline recipes/skore/meta.yaml:17]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
  entry_points:
    - skore = skore.__main__:main
```

**Comment by glemaitre** [comment]
@conda-forge/help-python, ready for review!

(Thanks in advance ;))

File: recipes/skore/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,65 @@
+{% set name = "skore" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "a24410ffb26b461c9b48d40c703242e4015b49ae01e73a91163d0df55d9100c1" %}
```
Comment (ocefpaf) [inline recipes/skore/meta.yaml:3]:
```suggestion
```

File: recipes/skore/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+{% set python_min = "3.9" %}
```
Comment (ocefpaf) [inline recipes/skore/meta.yaml:4]:
```suggestion
```

File: recipes/skore/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
Comment (ocefpaf) [inline recipes/skore/meta.yaml:17]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

**Review by ocefpaf — COMMENTED** [review]
Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

See https://github.com/conda-forge/staged-recipes?tab=readme-ov-file#grayskull---recipe-generator-for-python-packages-on-pypi for more information.

**Comment by glemaitre** [comment]
> Please use grayskull to generate Python recipes. Grayskull make it easier for us to review.

I updated the recipe using grayskull.

**Comment by glemaitre** [comment]
Thanks @ocefpaf

**Pull request merged**